Makefile

```diff
@@ -160,3 +160,15 @@ codespell: CODESPELL_BIN := codespell
 codespell:
 	which $(CODESPELL_BIN) >/dev/null 2>&1 || (echo "$(CODESPELL_BIN) binary not found, skipping spell checking"; exit 0)
 	$(CODESPELL_BIN) --skip $(CODESPELL_SKIP) --ignore-words .codespell.ignorewords --check-filenames --check-hidden
+
+.PHONY: install-controller
+install-controller:
+	CGO_ENABLED=0 GO111MODULE=on go build \
+		-o=lokomotive-controller \
+		-mod=$(MOD) \
+		-ldflags $(LDFLAGS) \
+		./cmd/lokomotive-controller
+
+.PHONY: docker-build-controller
+docker-build-controller:
+	docker build -f Dockerfile -t kinvolk/mutating-admission-controller .
```

Comment:
I'd put `kinvolk/mutating-admission-controller` in the variable.

Also, explicit `-f Dockerfile` is not needed IMO.